.github/workflows/build_with_conda.yml

```diff
@@ -53,6 +53,7 @@ jobs:
         env:
           ARCTICDB_USING_CONDA: 1
           ARCTICDB_BUILD_CPP_TESTS: 1
+          ARCTICDB_PROTOC_VERS: "4"
```

Comment:
I chose to restrict the compilation to protobuf 4 since this is the one picked with the specification of `environment.yaml`.